### DISCUSSION TITLE
Use Reanimated to create Animated Map Components and provide as a part from library

### DISCUSSION THREAD

**Discussion by ashuvssut** [react-native-maps/react-native-maps#4657] [Feature requests]
I [created animated components like Animated Map Circle & Animated Heading Marker](https://twitter.com/ashuvssut/status/1639327456015286285?s=20) using `react-native-reanimated`
I thought it would be nice if `react-native-maps` provide these components optionally. Developers who need these animated components will have to add Reanimated library in their dependencies to use these Animated components from `react-native-maps`

**Comment by MasterSigmar** [comment]
so euhhhhhhhhhhhhhhhhh where code ????
